File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.java
class Solution {
    public int solution(int[] nums) {
        int answer = 0;
        int sum = 0;
        for(int i = 0; i < nums.length-2; i++){
            for(int j = i+1; j < nums.length-1; j++){
                for(int k = j+1; k < nums.length; k++){
                    sum = nums[i] + nums[j] + nums[k];
                    int cnt = 0;
                    for(int n = 1; n < Math.round(Math.sqrt(sum))+1; n++){
                        if(sum % n == 0){
                            cnt++;
                        }
                    }
                    if(cnt <= 1){
                            answer++;
                        }
                }
            }
        }

        return answer;
    }
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
import itertools
import math

def primenumber(x):
    for i in range(2, int(math.sqrt(x)+1)):
    	if x % i == 0:	
        	return False
    return True	

def solution(nums):
    answer = 0
    npr = itertools.combinations(nums, 3)
    for a, b, c in list(npr):
        if primenumber(a+b+c):
            answer+=1
    return answer